src/backend/src/common/helpers/date-helper.ts

```diff
@@ -0,0 +1,10 @@
+export const toPacific = (date = new Date(), options = {}) => {
+  return new Date(date).toLocaleString('en', {
+    timeZone: 'America/Vancouver',
```

Comment:
probably a dumb question: do we need to handle daylight savings times whatsoever here?

Reply:
just wanted to make sure. It's come up in other projects.